orttraining/orttraining/python/training/torchdynamo/ort_backend.py

```diff
@@ -488,14 +502,15 @@ class OrtBackend:
         3. Inside _ort_accelerated_call, it creates onnxruntime.InferenceSession and calls it to execute the sub-graph.
     """
 
-    def __init__(self, ep: str = "", preallocate_output: bool = False):
+    def __init__(self, ep: str = "", preallocate_output: bool = False, session_options = None):
```

Comment:
Add document and a test for `session_options`. From the PR's message, looks like there should be a test for registering custom op in ORT.

Reply:
Thanks, Wei-Sheng. Added a test for custom op registration and execution via DORT